### DISCUSSION TITLE
After upgrade to 2.2, no clock or date in status bar.

### DISCUSSION THREAD

**Discussion by nrvale0** [regolith-linux/regolith-desktop#807] [Q&A] [answered]
Howdy. After a recent upgrade from 2.0 (I think) to 2.2 on which coincided with an upgrade of Ubuntu 22.04 to 22.10 I am no longer seeing a clock or date information in the status bar. 

```
$ regolith-diagnostic | egrep -i -e version -e clock
--- /etc/regolith/version:
REGOLITH_VERSION=2.2
Binary i3 version:  4.21 (2022-09-21) © 2009 Michael Stapelberg and contributors
Running i3 version: 4.21 (2022-09-21) (pid 4836)
org.gnome.desktop.interface clock-format '12h'
org.gnome.desktop.interface clock-show-date true
org.gnome.desktop.interface clock-show-seconds false
org.gnome.desktop.interface clock-show-weekday false
No such schema “org.regolith-linux.ilia”
```

My ~/.config/regolith2 is not very extensive. A quick perusal of /usr/share/regolith and /usr/share/i3xrocks doesn't yield anything suspicious. 

Any thoughts on why that might be?

**Comment by nrvale0** [comment]
For some reason the i3xrocks-time package was not installed after the upgrade. Adding this package resolved the issue.

**Comment by kgilmer** [comment]
@nrvale0 , do you happen to be experiencing the same issue?

https://github.com/regolith-linux/regolith-control-center/issues/8

**Reply by nrvale0** [reply]
Sorry, its been so long ago I cannot recall much about the problem nor the fix other than what is posted here.

**Comment by mauromotion** [comment]
Same here, but the package i3xrocks-time can't be installed on Pop!_OS 22.04. See output:

```sudo apt install i3xrocks-time
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies.
 regolith-control-center : Depends: libadwaita-1-0 (>= 1.2~alpha) but 1.1.0-1ubuntu2 is to be installed
                           Depends: libcolord-gtk4-1 (>= 0.1.24) but it is not installable
                           Depends: libgnome-bg-4-2 (>= 3.27.90) but it is not installable
                           Depends: libgnome-desktop-4-2 (>= 3.33.4) but it is not installable
                           Depends: libgnome-rr-4-2 (>= 3.17.92) but it is not installable
                           Depends: libgoa-1.0-0b (>= 3.45) but 3.40.1-2ubuntu1pop0~1649081188~22.04~8042754 is to be installed
                           Depends: libnma-gtk4-0 (>= 1.8.34) but it is not installable
                           Depends: gnome-control-center-data (>= 1:43.0) but 1:41.4-1ubuntu12pop1~1653075611~22.04~64c84a2 is to be installed
                           Recommends: gnome-bluetooth-sendto but it is not installable
                           Recommends: gnome-user-share but it is not going to be installed
                           Recommends: power-profiles-daemon but it is not going to be installed
                           Recommends: malcontent-gui but it is not going to be installed
                           Recommends: libnss-myhostname but it is not going to be installed
                           Recommends: pulseaudio-module-bluetooth but it is not going to be installed
                           Recommends: realmd but it is not going to be installed
E: Unable to correct problems, you have held broken packages.```

**Reply by kgilmer** [reply]
I cannot say for sure but it's possible that pop 22.04 should map to Ubuntu `kinetic` packages rather than `focal`.  It may work if you update the Regolith apt url to point to the kinetic repo.

`/etc/apt/sources.list.d/regolith.list`:

```
deb [arch=amd64 signed-by=/usr/share/keyrings/regolith-archive-keyring.gpg] https://regolith-desktop.org/release-ubuntu-kinetic-amd64 kinetic main
```

**Comment by nrvale0** [comment]
I recently went back to Pop!_OS 22.04 from Ubuntu 22.10 and I am able to repro the missing clock as well as the i3xrocks-time package being uninstallable as detailed by @mauromotion. Also, the sources list is pointing to kinetic:

```
deb [arch=amd64 signed-by=/usr/share/keyrings/regolith-archive-keyring.gpg] https://regolith-desktop.org/release-ubuntu-kinetic-amd64 kinetic main
```

@kgilmer

**Comment by nrvale0** [comment, marked as answer]
Umm...Pop!_OS 22.04 is based off of Ubuntu 22.04 == jammy. When I changed the installer-generated `/etc/apt/sources.list.d/regolith.list` from `kinetic` to `jammy` and did:

    sudo apt remove --purge -y regolith*
    sudo apt install regolith-desktop regolith-compositor-<preferred>

and then did a config reload and i3 reload everything loaded cleanly including the clock.

**Comment by randunel** [comment]
Since this is the first google result when regolith stops showing the time following an upgrade, make sure you have `i3xrocks-time` installed, first.

```
sudo apt install i3xrocks-time
```
